tests/config.env

```diff
@@ -0,0 +1,6 @@
+OSC_TEST_LOGIN="user@domain.tld"
+OSC_TEST_PASSWORD="P4ssW0rd"
+OSC_TEST_ACCESS_KEY="S65FD45EXAMPLE"
```

Comment:
I would make it even more explicit that this is an example using something like:
```
OSC_TEST_ACCESS_KEY="3X4MPL3EXAMPLE" 
OSC_TEST_ACCESS_KEY="3X4MPL3EXAMPLE3X4MPL3EXAMPLE3X4MPL3"
```